content/blog/2025/251028-stac-sprint-eu-recap.md

```diff
@@ -0,0 +1,69 @@
+---
+date: 2025-10-25T08:00:00-00:00
+title: "STAC+Zarr Community Sprint 2025 Recap"
+author: "[Emmanuel Mathot](https://www.linkedin.com/in/emmanuelmathot/),[Julia Signell](https://www.linkedin.com/in/jsignell/), & [Michelle Roby](https://www.linkedin.com/in/the-michelle-roby/)"
+author_title: "STAC PSC Members"
+summary: "The first European STAC sprint brought together developers at ESA ESRIN to establish practical patterns and best practices for integrating STAC with Zarr for cloud-native multidimensional geospatial data."
+---
+
+From October 14-16, the STAC and Zarr communities came together at ESA ESRIN in Frascati, Italy, for the first-ever STAC sprint in Europe focused on advancing cloud-native multidimensional geospatial data. After three intensive days of collaborative development, we're excited to share what we accomplished and what we hope to be the path forward for STAC-Zarr integration.
+{{< img src="images/251025-stac-sprint-group-photo.jpg" caption="Participants gathered at ESA ESRIN for the first European STAC sprint">}}
+
+## A Community Comes Together
+
+Twenty-six developers from across Europe and beyond converged on ESA's facility in Frascati, representing a remarkable cross-section of the geospatial data community. Organizations including ESA, CEDA, DLR, EODC, Eurac Research, CloudFerro, Terradue, Element 84, Development Seed, DKRZ, Tilebox, and many others brought their unique perspectives and expertise to hammer out how multidimensional data should be represented in STAC.
+
+What made this sprint particularly powerful was the convergence of two communities—STAC and Zarr—working together on shared problems. For several years there have been discussions and ideas floating around about how to use STAC with Zarr, but getting everyone together, and being able to look at some existing implementations proved incredibly fruitful.
+
+The sprint focused on fundamental questions that STAC faces as it gains popularity outside of the Earth Observation community: How do you represent multidimensional Zarr stores in STAC? What's the best way to handle variables and dimensions for datacubes? How should we navigate hierarchical Zarr structures? These aren't theoretical exercises—they're the real challenges faced by organizations working with petabytes of Earth observation data.
+
+## Key Achievements
+
+### Specification Work
+
+The sprint produced several concrete outputs that will shape how we work with multidimensional geospatial data:
+
+- **[Best Practices for STAC Zarr and N-Dimensional Arrays](https://github.com/radiantearth/stac-best-practices/pull/29)**: A comprehensive draft providing guidance on using Zarr with STAC. This is significant because, until now, nobody had proper guidance on this integration. The PR is ready for community review and needs feedback to refine and finalize the recommendations. We would like to merge it soon to provide clarity to implementers.
```

Comment:
Maybe provide something more specific than "soon" so that people actually have a deadline for review.

Reply:
I added an invitation to comment or participate in CM